src/components/GenericModal.vue

```diff
@@ -137,7 +137,7 @@ export default {
   },
   mounted() {
     this.focusTrapInstance = new FocusTrap();
-    document.addEventListener('keydown', this.onEscapeClick);
+    document.addEventListener('keydown', this.onClick);
```

Comment:
Should we rename `onClick` to something else since a "click" is never involved with these events? Maybe `onKeyDown` or something?

Reply:
done! https://github.com/apple/swift-docc-render/pull/39/commits/b3e5bb8643c9e34efcd8c3350bde4a4d06b4522b